fix-generated.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Patch types.go to fix names for enum constants. This is a workaround for https://github.com/deepmap/oapi-codegen/issues/1614.
+file=openapi/types.go
+sed 's/\([A-Za-z0-9_]*\)\(  *\)\(ProjectStatusModelState\) = "\1"/\3\1\2\3 = "\1"/' "$file" > "${file}.tmp"
```

Comment:
Do we need the same fix for the `DatabaseStatusModelState` and `BackupStatusModelState`?

Reply:
No, it is only happening for the `ProjectStatusModelState`.